jobs/WebUi/UnitTest/UnitTest.groovy

```diff
@@ -0,0 +1,6 @@
+def UnitTest = load("jobs/UnitTest/UnitTest.groovy")
```

Comment:
I think this implementation.
but consider if the UnitTest.groovy runs outside on-build-config repo.